src/screens/PutawayList/index.tsx

```diff
@@ -144,13 +144,23 @@ class PutawayList extends React.Component<Props, State> {
     });
   };
   onPutAwayTapped = (putAway: PutAway) => {
-    this.props.navigation.navigate('PutawayDetail', {
-      // putAway,
-      putAway: putAway,
-      exit: () => {
-        this.props.navigation.navigate('PutawayList');
-      },
-    });
+    console.log('putAway ::', putAway);
+    if (putAway?.putawayItems?.length > 1) {
+      this.props.navigation.navigate('PutawayDetail', {
+        putAway: putAway,
+        exit: () => {
+          this.props.navigation.navigate('PutawayList');
+        },
+      });
+    } else {
+      this.props.navigation.navigate('PutawayItemDetail', {
+        putAway: putAway,
+        putAwayItem: putAway?.putawayItems[0],
+        exit: () => {
```

Comment:
Do it as: `exit: () => this.props.navigation.navigate('PutawayList')`

Reply:
@awalkowiak it's an IDE auto formation. it's automatically added into the next line and I don't think is there has any issue with line formations.